server/lib/src/constants/entries.rs

```diff
@@ -642,623 +643,526 @@ impl EntryClass {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
 /// Built-in group definitions
 pub struct BuiltinGroup {
     pub name: &'static str,
     description: &'static str,
-    classes: Vec<EntryClass>,
     uuid: uuid::Uuid,
-    member: uuid::Uuid,
+    members: Vec<uuid::Uuid>,
+    dyngroup: bool,
```

Comment:
I think dyngroup should be seperate to BuiltinGroup since a built in dyngroup should have *no* members hardcoded.

Reply:
I was trying to limit the number of new types to define, especially since later we're doing a Vec<BuiltinGroup> to build the schema, which could lead to some messy dependencies around which order things get added to the database - I've added a runtime check creating these groups with members defined.

Reply:
I think that's a reasonable compromise.